_data/Ghamry0x1/ghamry0x1.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Mohamed ELGHAMRY # No longer than 28 characters
+institution: Faculty of Engineering, Ain Shams University 🚩 # no longer than 58 characters
+profile_pic: ghamry0x1.png # Name and extension of your profile picture(ex. mona.png)
```

Comment:
Your profile picture extension does not match the file (image is a ```.jpg```, not a ```.png```).

Reply:
Got it!
I've created a new pull request as the repo was deleted by mistake. Here's the [PR URL](https://github.com/education/graduation/pull/3811)